backend/tests/routes/test_nylas_service.py

```
@@ -0,0 +1,101 @@
+import pytest
+from unittest.mock import patch, MagicMock
+
+from backend.app.api.routes.nylas import send_notification_email, get_active_grant_id
```

Ruff (0.8.2)

`backend.app.api.routes.nylas.get_active_grant_id` imported but unused

Remove unused import: `backend.app.api.routes.nylas.get_active_grant_id`

(F401)